webrtc/RTCPeerConnection-createDataChannel.html

```diff
@@ -3,44 +3,51 @@
 <title>RTCPeerConnection.prototype.createDataChannel</title>
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
+<script src="RTCPeerConnection-helper.js"></script>
 <script>
 'use strict';
 
-// Test is based on the following editor draft:
-// https://rawgit.com/w3c/webrtc-pc/cc8d80f455b86c8041d63bceb8b457f45c72aa89/webrtc.html
+// Test is based on the following revision:
+// https://rawgit.com/w3c/webrtc-pc/1cc5bfc3ff18741033d804c4a71f7891242fb5b3/webrtc.html
```

Comment:
I am not a big fan of these comments, they will become obsolete at some point without anyone noticing.